test/jobs/zendesk_job_test.rb

```diff
@@ -0,0 +1,61 @@
+require_relative '../test_helper'
+
+SingleCov.covered!
+
+describe 'Zendesk Job' do
+  hostname = 'https://hostname.com'
+  username = 'username@email.com'
+  password = '123456'
+  describe 'boostraps zendesk job' do
+    it 'bootstraps by getting environment variables' do
+      ENV['ZENDESK_HOSTNAME'] = hostname
+      ENV['ZENDESK_USERNAME'] = username
+      ENV['ZENDESK_PASSWORD'] = password
+      ZendeskJob.bootstrap()
+      assert ZendeskJob.hostname == hostname
+      assert ZendeskJob.username == username
+      assert ZendeskJob.password == password
+    end
+  end
+  
+  describe 'has zendesk integration' do
+    it 'should return true if zendesk variables are set' do
```

Comment:
avoid `should` and `return`. `should` is a reference to an old testing style and is implied, and everything will return.